lib/fsi.go

```diff
@@ -27,6 +27,22 @@ func (m *FSIMethods) Name() string {
 	return "fsi"
 }
 
+// Mapping is FIXME
+func (m *FSIMethods) Mapping() map[string][]string {
+	return map[string][]string{
```

Comment:
As hard coded as it is, I don't see a better way around it. Hence having it centralized is great for the future 👍

Reply:
Sounds good. One thing to keep in mind, thinking far into the future: I've been envisioning this mapping more as a way to keep "metadata" about the methods in question. For example, maybe at some point we also add some kind of permissions information, or multi-processing info (is this method same to call in parallel with others?), so it may expand to be about more than just HTTP. Totally possible that I'm planning too far ahead with that idea, so I'll switch to your suggestion, but we may need to reevaluate in the future.

Reply:
ah ok, then yes let's drop the `HTTP`, but that's also  more reason to use a struct instead of a string slice